website/content/v1.10/talos-guides/configuration/image-cache.md

```diff
@@ -121,6 +121,15 @@ If the disk image is used, the `IMAGECACHE` volume doesn't need to be configured
 
 See [disk management]({{< relref "./disk-management#machine-configuration" >}}) for more information on volume configuration.
 
+## Updating the Cache
+
+The image cache is populated during installation from the boot media and stored on disk.
+
+Once Talos is running, this cache becomes read-only and cannot be updated on a live node.
```

Comment:
why is that? do you have this tested that it doesn't work?

Reply:
That comment is very old, I haven't tried, but I do believe if you mount an ISO/USB stick which looks like an Image Cache, modern Talos should copy its contents over in live mode (for a running machine).